plankton/core/utils.py

```diff
@@ -25,39 +25,78 @@
 from __future__ import absolute_import
 from six import string_types
 
-import imp
 import importlib
 import textwrap
 import inspect
-from datetime import datetime
 
-import os.path as osp
+from os import path as osp
 from os import listdir
 
+from datetime import datetime
+
 from .exceptions import PlanktonException
 
 
-def get_available_submodules(package):
+def get_submodules(module):
```

Comment:
You're right in that they are not used elsewhere, but they are fairly general. I wouldn't want to put them into `devices` mainly because of their very general nature...maybe we can find a better place than `utils` at some point, but for now I'd rather leave them there.